### PR TITLE
Scaladoc: Add member position IDs to anchors in searchbar

### PR DESCRIPTION
**Pull request by jchyb**
This PR includes following changes, each accompanied by a separate commit:
* id location attributes were added to anchors in searchbar - this way, even if a member is located 
* searchbar will now always be closed after clicking a result - this is to keep behavior of the search result being located in a separate URL (where searchbar would always be closed) and in a different location of the same URL (where, previously, searchbar would stay open, even after snapping to a different location).
* Linking to the internal docs via Inkuire was fixed - it felt related to the other two changes, so it's included in this PR (but I can split it up if necessary).

### CODE REVIEW

**Comment by KacperFKorban** [comment]
https://scala3docstorage.blob.core.windows.net/pr-searchbar-anchors/scala3/api/index.html

File: scaladoc/src/dotty/tools/scaladoc/tasty/InkuireSupport.scala [hunk 1]
```diff
@@ -13,10 +14,13 @@ import SymOps._
 import NameNormalizer._
 import SyntheticsSupport._
 
-trait InkuireSupport:
+trait InkuireSupport(using DocContext) extends Resources:
   self: TastyParser =>
   import qctx.reflect._
 
+  // Unused in InquireSupport, required for Resources
```
Comment (KacperFKorban) [inline scaladoc/src/dotty/tools/scaladoc/tasty/InkuireSupport.scala:21]: a small typo `InkuireSupport`

(K was for Kotlin back then :smile: )

**Review by KacperFKorban — APPROVED** [review]
Looks good to me. Great job :smiley:

**Comment by pikinier20** [comment]
Good job!

**Pull request merged**